packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.html

```diff
@@ -80,8 +80,7 @@ <h2 class="heading_underline" data-translate="sms_message_title"></h2>
                 <oui-checkbox
                     name="noCommercialClause"
                     id="noCommercialClause"
-                    model="$ctrl.model.noStopClause"
-                    on-change="$ctrl.getMessageDetails(modelValue)"
```

Comment:
No more calculation made after checkbox changed ?

Reply:
from what i understand only the noStopClause does change the calculation, am i missing something ?

Reply:
Oh I didn't saw you change from `noStopClause` to `noCommercialClause`, my bad, it's ok then